CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 Prefix your items with `(Template)` if the change is about the template and not the resulting application.
 
+## 4.0.X
+
+### Security
```

Comment:
You don't need to add subheaders, nor notes. As you can see, we've already done security updates in the past (look at the mention of `System.Text.Json`).